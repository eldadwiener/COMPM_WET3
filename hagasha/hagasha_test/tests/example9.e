COPYI I1 0
COPYI I2 0
COPYI I3 0
COPYI I4 0
COPYI I5 0
COPYI I6 0
COPYI I7 0
JLINK 10
HALT
PRNTC 84
PRNTC 101
PRNTC 115
PRNTC 116
PRNTC 58
PRNTC 32
PRNTC 101
PRNTC 115
PRNTC 99
PRNTC 97
PRNTC 112
PRNTC 101
PRNTC 100
PRNTC 32
PRNTC 99
PRNTC 104
PRNTC 97
PRNTC 114
PRNTC 97
PRNTC 99
PRNTC 116
PRNTC 101
PRNTC 114
PRNTC 115
PRNTC 58
PRNTC 32
PRNTC 40
PRNTC 110
PRNTC 101
PRNTC 119
PRNTC 108
PRNTC 105
PRNTC 110
PRNTC 101
PRNTC 41
PRNTC 10
PRNTC 98
PRNTC 97
PRNTC 99
PRNTC 107
PRNTC 115
PRNTC 108
PRNTC 97
PRNTC 115
PRNTC 104
PRNTC 58
PRNTC 32
PRNTC 92
PRNTC 44
PRNTC 32
PRNTC 113
PRNTC 117
PRNTC 111
PRNTC 116
PRNTC 101
PRNTC 115
PRNTC 58
PRNTC 32
PRNTC 34
PRNTC 44
PRNTC 32
PRNTC 40
PRNTC 110
PRNTC 101
PRNTC 119
PRNTC 108
PRNTC 105
PRNTC 110
PRNTC 101
PRNTC 41
PRNTC 10
COPYI I2 I1
RETRN
HALT
