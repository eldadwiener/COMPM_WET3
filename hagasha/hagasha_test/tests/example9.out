Test: escaped characters: (newline)
backslash: \, quotes: ", (newline)
Reached Halt.
